### ISSUE TITLE
How to train my own model using my own data

### ISSUE THREAD

**Issue by huayong** [torrvision/crfasrnn#2]
Hi bittnt,

I had already installed crfasrnn and the code worked well.
But the code that you provided just contained the part using the trained model (TVG_CRFRNN_COCO_VOC.caffemodel) to get the result. How could I train a new model using my own data. Could you provide a pipeline about how to train a new model and the prototxt file used in training.
Any response will be appreciated.

Best wishes,
Huayong

**Comment by bittnt** [comment]
@huayong 

we describe the training procedure in our ICCV 2015 paper (http://www.robots.ox.ac.uk/~szheng/papers/CRFasRNN.pdf). The parameters of the solver file for pascal voc dataset can be found in the paper. The prototxt and model are provided. The learning rate for each component of crf-rnn layer is given. To train the model on new dataset,  you only need to set up your own solver file. 

Thanks.

**Comment by mtourne** [comment]
It would be great if the code you used for it was provided as well :)

**Comment by cracker0dks** [comment]
:+1: would be nice

**Comment by thuanvh** [comment]
I am trying to train my own images. I wonder how to create the training prototxt.
How do you use the label input? Which layer that you used for the loss calculation?

Thanks.

**Comment by thesby** [comment]
I want to train my own model based on this model.  Any one can provide a train_val.prototxt?  Thank you.

**Comment by sadeepj** [comment]
Closing old issues with no recent activity.
